tpl/page_optm/settings_html.tpl.php

```diff
@@ -87,6 +87,26 @@
 			</td>
 		</tr>
 
+		<tr>
+			<th>
+				<?php $id = Base::O_OPTM_HTML_SKIP_COMMENTS; ?>
+				<?php $this->title($id); ?>
+			</th>
+			<td>
+				<?php $this->build_textarea($id); ?>
+				<div class="litespeed-desc">
+					<?php echo __('When minifying HTML do not discard comments that match a specified pattern.', 'litespeed-cache'); ?>
+					<br />
+					<?php echo __('If comment to be kept is like:', 'litespeed-cache'); ?>
+					<code>&lt;!-- A comment that needs to be here --&gt;</code>
+					<?php echo __('write:', 'litespeed-cache'); ?>
```

Comment:
You are almost there. Here it will be more straightforward for translation if keep 100 and 102 in same line as they are in same sentence context.

Reply:
combined all together